electrumsv/gui/qt/network_dialog.py

```diff
@@ -870,8 +870,10 @@ def _event_clicked_button_save(self) -> None:
         assert self._is_form_valid(), "should only get here if the form is valid and it is not"
 
         server_type = self._get_server_type()
-        # We normalise the url and remove the suffix to ensure less change of duplicates.
-        server_url = self._server_url_edit.text().strip().lower().removesuffix("/")
+        # All server urls must end with "/"
```

Comment:
This comment just describes the what and not the why. Why even have a comment if it just puts in words literally what the code does? Please add back the old comment corrected.